internal/controller/config.go

```diff
@@ -120,6 +121,32 @@ func (r *ConfigReconciler) createConfigData(
 		data[bucket.EnvVar] = bucket.AccessPointARN
 	}
 
+	type PVCMap struct {
+		PVCName       string `json:"pvcName"`
+		RootDirectory string `json:"rootDirectory"`
+	}
+	var pvcMaps []PVCMap
+	for _, pvc := range spec.Storage.PersistentVolumeClaims {
+		for _, pv := range spec.Storage.PersistentVolumes {
+			if pv.Name == pvc.PVName {
+				for _, ap := range spec.AWS.EFS.AccessPoints {
+					if ap.Name == pv.VolumeSource.AccessPointName {
+						pvcMaps = append(pvcMaps, PVCMap{
+							PVCName:       pvc.Name,
+							RootDirectory: ap.RootDirectory,
+						})
+						break
+					}
```

Comment:
After discussion, this will be updated to save the PVName to the config map, so that Pycalrissian can then construct the expected mount path: `/workspace/<block-store-name>`

Reply:
Changed to store PVCName and PVName. Pycalrissian constructs the path.

Reply:
Perfect!